### PR TITLE
Add additional visitor spec

### PR DESCRIPTION
**Pull request by EGiataganas**

### CODE REVIEW

File: spec/models/additional_visitor_spec.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe AdditionalVisitor do
+  it { should belong_to(:visit) }
```
Comment (dan-palmer) [inline spec/models/additional_visitor_spec.rb]: We should use RSpec's `expect` syntax here:

```ruby
it { is_expecetd.to belong_to :visit }
```

Comment (dan-palmer) [inline spec/models/additional_visitor_spec.rb]: Same applies to the other tests. This is worth a read: http://rspec.info/blog/2012/06/rspecs-new-expectation-syntax/

Comment (SteveMarshall) [inline spec/models/additional_visitor_spec.rb]: @dan-palmer `is_expecetd`, eh? Is that a new feature? :trollface:

Comment (dan-palmer) [inline spec/models/additional_visitor_spec.rb]: 😭

**Comment by SteveMarshall** [comment]
@threedaymonk LGTM! Rebase?

**Pull request merged**